dds/DCPS/DataReaderImpl_T.h

```diff
@@ -1,6 +1,12 @@
 #ifndef OPENDDS_DCPS_DATAREADERIMPL_T_H
 #define OPENDDS_DCPS_DATAREADERIMPL_T_H
 
+#include "ace/config-lite.h"
+
+#ifdef ACE_HAS_CPP11
+#  define HAS_STD_SHARED_PTR
```

Comment:
Macros need to start with OPENDDS

Reply:
OK. I was following the example found in unique_ptr.cpp which also lacks OPENDDS. Should I modify that file as well?

Reply:
That's up to you.  We can either do it now, or note it and come back to it later.